Test cases/strings.swift

```diff
@@ -98,3 +98,10 @@ for index in abc.indices {
 for char in abc {
 	print(char)
 }
+
+print("$foo")
+print("${foo}")
+
+let bar = "$foo"
+let  baz = "\(bar)"
+print(baz)
```

Comment:
Could you add a line here with an interpolated string? Something like `print("${foo} \(bar)")`, just to be sure we don't break that accidentally in the future.

Also, minor nitpick, I think there's an extra space before `baz`?